src/App/mermaidData/pullApiData.js

```diff
@@ -45,14 +42,6 @@ export const pullApiData = async ({
     await getAuthorizationHeaders(getAccessToken),
   )
 
-  // If projects are part of the API pull determine if the user has been
```

Comment:
this is now taken care by deleting whatever is in the `removes` array that the api sends as part of the pull response